runtime/compiler/runtime/JITServerAOTCache.hpp

```diff
@@ -72,6 +72,17 @@ struct AOTCacheClassChainRecord;
 struct AOTCacheWellKnownClassesRecord;
 struct AOTCacheAOTHeaderRecord;
 
+struct JITServerAOTCacheReadContext
```

Comment:
This struct is not part of the public API, its definition should be moved to the .cpp file.